.azuredevops/templates/DirectXTex-build-gdk.yml

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+#
+# http://go.microsoft.com/fwlink/?LinkId=248926
+
+parameters:
+- name: msVersion
+  type: string
```

Comment:
In YAML a 'number' is apparently an integer, or at least it shortens ``17.0`` to ``17`` which makes MSBuild complain.